.github/workflows/ci_linux_x86_clang.yml

```diff
@@ -0,0 +1,67 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: CI - Linux x86 clang
```

Comment:
is this actually a 32-bit build? (x86 is 32-bit, x64 or x86_64 are 64-bit)

Reply:
One of these days I'll get that right, lol. It's 64 bit yeah.

Reply:
So `.github/workflows/ci_windows_x86_msvc.yml` should be `x64` too?

Reply:
Yeah - `-- Found assembler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.40.33807/bin/Hostx64/x64/cl.exe` https://github.com/iree-org/iree/actions/runs/10056373165/job/27795028422#step:7:131

Reply:
Same for macOS: `++ IREE_PYTHON3_EXECUTABLE=/Users/runner/hostedtoolcache/Python/3.10.14/x64/bin/python3` https://github.com/iree-org/iree/actions/runs/10056373697/job/27795030174#step:8:31